src/pages/Articles.jsx

```diff
@@ -1,13 +1,40 @@
+import { useEffect, useState } from "react"
 import { useSearchParams } from "react-router-dom"
+import axios from "axios"
+import ArticleTile from "../components/ArticleTile"
 
 export default function Articles () {
     const [searchParams, setSearchParams] = useSearchParams()
 
+    const [articles, setArticles] = useState([])
+
+
     const topic = searchParams.get('topic')
-    console.log(topic)
+
+    useEffect(() => {
+        const getArticles = async () => {
+            try {
+                let url = `https://nc-news-p6kr.onrender.com/api/articles`
+                const response = await axios.get(url)
+                setArticles(response.data.articles)
+            }catch(err) {
+                console.log(err)
```

Comment:
you can refactor to use baseURL set up using axios.create and you should be exporting func to get all articles